content/working-with-css.md

```diff
@@ -0,0 +1,46 @@
+# Way of working with CSS
+
+## CSS, Sass or SCSS?
+SCSS is preferred, this enables the use of variables for media queries and it enables nesting.
+
+## Nesting
+Nesting is allowed for two levels and only for modifiers like hover states and pseudo elements.
+
+### Use of `&`
+The Ampersand symbol `&`  can only be used in these cases:
+- Nesting classes on the same element:
+```css
+.class {
```

Comment:
This should never happen, you should always utilize BEM. Not even give this example, because we work BEM

Reply:
I think there can be cases where an element has multiple class names, not only modifiers. Although not preferable, I think it can be allowed to use it like this.